src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -55,7 +55,7 @@ public function __construct(array $configuration, array $connectionCredentials =
         $this->connection->connect($connectionCredentials['host'] ?? '127.0.0.1', $connectionCredentials['port'] ?? 6379);
         $this->connection->setOption(\Redis::OPT_SERIALIZER, $redisOptions['serializer'] ?? \Redis::SERIALIZER_PHP);
 
-        if (isset($connectionCredentials['auth']) && !$this->connection->auth($connectionCredentials['auth'])) {
+        if (!empty($connectionCredentials['auth']) && !$this->connection->auth($connectionCredentials['auth'])) {
```

Comment:
I would avoid using `empty` (`empty('0') === true`).

Reply:
Oh good catch. 

I changed it to:

```php
$auth = $connectionCredentials['auth'] ?? null;
if ($auth && !$this->connection->auth($auth)) {
```